Rakefile

```diff
@@ -87,6 +87,18 @@ def generate_carthage_project
   sh "mint run yonaskolb/xcodegen@#{XCODEGEN_VERSION} 'xcodegen --spec carthage-project.yml'"
 end
 
+def osx
+  if ENV["TRAVIS_OS_NAME"] == "linux"
+    return :linux
+  else
```

Comment:
Why not check if os name is `osx` like in `.travis.yml`?

Reply:
Good idea @allu22. I've updated it.